lessons/beginners/functions/index.md

```diff
@@ -283,8 +283,8 @@ Výraz `len(s)` je *volání funkce*, ale `len` bez závorek označuje
 
 Výsledek `len(s)` je číslo; `len` je funkce.
 
-Čísla můžeš sečítat, můžeš tedy napsat `len(s) + 1`.
-Funkce ale sečítat nejde – `len + 1` nedává smysl.
+Čísla můžeš sčítat, můžeš tedy napsat `len(s) + 1`.
+Funkce ale sčítat nejde – `len + 1` nedává smysl.
```

Comment:
Mám za to že obojí je správně: https://www.pravidla.cz/pravopis/19653-secitat.html
Nebo to je jinak?

Reply:
Aha, pardon. Příště se mrknu. :)